Libraries/Contents/ContentExtensions.cs

```diff
@@ -128,5 +128,15 @@ public static async Task SanitizeContentItemVersionsAsync(this IContent content,
         /// <param name="content">Content item containing <see cref="AliasPart"/>.</param>
         /// <returns>Alias of the content item.</returns>
         public static string GetAlias(this IContent content) => content.As<AliasPart>()?.Alias;
+
+        /// <summary>
+        /// Provides the most essential data for a <see cref="ContentItem"/> enough to identify it in a text format. Can
+        /// be used as a human-readable text representing the <see cref="ContentItem"/> in a log.
+        /// </summary>
+        /// <returns>Technical text representing a Content Item.</returns>
+        public static string ToTechnicalString(this IContent content) =>

```

Comment:
Ez nem tudom miért van itt. Lehet, hogy mergeltem valamit amit nem kéne, de nem revertezni mert a múltkoritól is kiborult utána a bili. Ha nem ok, akkor inkább nyitok egy új branchot és beillesztem oda a lenti `Any`-t

Reply:
Azért, mert a https://github.com/Lombiq/Helpful-Libraries/pull/27-ot elfelejtettem bemergelni.